code/modules/clothing/spacesuits/ert.dm

```diff
@@ -16,15 +16,26 @@
 		"Vox" = 'icons/mob/species/vox/helmet.dmi'
 		)
 
+/obj/item/clothing/head/helmet/space/hardsuit/ert/Initialize()
+	var/mob/living/carbon/human/wearer = src.loc.loc	//src.loc is the hardsuit, so its loc is the wearer
```

Comment:
Best to do a null check on this.
It is "possible" that this is not true
`?.` can be used in this case

Reply:
If it fails, ishuman will just return 0, won't it?

Reply:
Yeah thinking about it. `src.loc` will prob be either the hardsuit or a turf. Then the loc of that will be the player, turf or area in the last case. So it works.
src is implied in this case though